test/LRPTests/APITests.cpp

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include <TestDef.h>
+#include <TlHelp32.h>
+#include <NotificationsLongRunningProcess_h.h>
+
+using namespace WEX::Common;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::System;
+
+namespace Test::LRP
+{
+    static const PCWSTR c_processName = L"TAEF.exe";
+    static const PCWSTR c_appId = L"toastAppId";
+    inline IID c_remoteId = winrt::guid("A7652901-313C-4EFA-A303-95C371A00DAB");
```

Comment:
Should the remote ID be renewed from time to time?

Reply:
This is a fake remoteId to fake LRP registrations so we don't have to renew this.